### PR TITLE
Feat/left side toolbar

### PR DESCRIPTION
**Pull request by GarryFirst-cripto**
Implement left-side chats toolbar UI.

### CODE REVIEW

File: frontend/src/containers/ChatToolbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+import { IChat } from '../../common/models/workstate/chat';
+import styles from './styles.module.sass';
+import { userChannelRoutine } from '../../scenes/Workspace/routines/routines';
+
+let linkPanel = false;
+let directPanel = false;
+
+interface IProps {
+  workstate: IWorkspaceState;
+  userChannelRoutine: Routine;
```
Comment (AlikRakhmonov) [inline frontend/src/containers/ChatToolbar/index.tsx:31]: see how it's implemented in other places

File: frontend/src/containers/ChatToolbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  };
+  const doFileBrowseSelect = (chat: IChat) => {
+    // Select File Browser
+    channelRoutine(chat);
+  };
+  const doShowLessSelect = (chat: IChat) => {
+    // Select Show less Channel
+    channelRoutine(chat);
+  };
+  const doSelectChannel = (channel: IChat) => {
```
Comment (AlikRakhmonov) [inline frontend/src/containers/ChatToolbar/index.tsx:72]: lots of code duplications

File: frontend/src/containers/ChatToolbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
+import React from 'react';
```
Comment (AlikRakhmonov) [inline frontend/src/containers/ChatToolbar/index.tsx:1]: move it to Workspace scene

File: frontend/src/containers/ChatToolbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+interface IProps {
+  workstate: IWorkspaceState;
+  userChannelRoutine: Routine;
+}
+
+const ChatToolbar = ({
+  workstate,
+  userChannelRoutine: channelRoutine
+}: IProps) => {
+  const { channels, directMessages, selectedChat } = workstate;
```
Comment (AlikRakhmonov) [inline frontend/src/containers/ChatToolbar/index.tsx:38]: you should map it in mapStateToProps

File: frontend/src/containers/ChatToolbar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+  );
+  const getClassNameImg = (state: boolean) => (
+    state
+      ? styles.chanelsImgRotate
+      : styles.chanelsImg
+  );
+
+  const changeListView = () => {
+    linkPanel = !linkPanel;
+    const element = document.getElementById('divChanel') as HTMLElement;
```
Comment (AlikRakhmonov) [inline frontend/src/containers/ChatToolbar/index.tsx:94]: it's incorrect to work with dom here. it's better to work with flags and styles

Comment (GarryFirst-cripto) [reply frontend/src/containers/ChatToolbar/index.tsx:94]: Done

File: frontend/src/containers/ChatToolbar/index.tsx (hunk 4 above)
Comment (GarryFirst-cripto) [reply frontend/src/containers/ChatToolbar/index.tsx:38]: Done

File: frontend/src/containers/ChatToolbar/index.tsx (hunk 2 above)
Comment (GarryFirst-cripto) [reply frontend/src/containers/ChatToolbar/index.tsx:72]: Done

File: frontend/src/common/models/workstate/chat.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export interface IChat {
+  id: string;
+  channelName: string;
```
[the remark this answers is not in the document]
Comment (GarryFirst-cripto) [reply frontend/src/common/models/workstate/chat.ts:3]: Done

File: frontend/src/containers/ChatToolbar/index.tsx (hunk 3 above)
Comment (GarryFirst-cripto) [reply frontend/src/containers/ChatToolbar/index.tsx:1]: Done

File: frontend/src/containers/ChatToolbar/index.tsx (hunk 1 above)
Comment (GarryFirst-cripto) [reply frontend/src/containers/ChatToolbar/index.tsx:31]: Done

**Pull request closed without merging**